src/Illuminate/Validation/Validator.php

```diff
@@ -1898,6 +1898,43 @@ protected function validateTimezone($attribute, $value)
     }
 
     /**
+     * Validate that an image is a valid orientation.
+     *
+     * @param  string  $attribute
+     * @param  mixed   $value
+     * @param  array   $parameters
+     * @return bool
+     *
+     * @throws \InvalidArgumentException
+     */
+    protected function validateOrientation($attribute, $value, $parameters)
+    {
+        if (! $this->isAValidFileInstance($value) || ! $sizeDetails = getimagesize($value->getRealPath())) {
+            return false;
+        }
+
+        if (empty($parameters)) {
+            throw new InvalidArgumentException('Validation rule orientation requires 1 parameter.');
+        }
```

Comment:
@themsaid I did have that in before, but replaced it with the `empty()` version in my service provider. I’ll revert back to the `requireParameterCount()` method instead.